plugins-list.yaml

```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
+plugins/notifications-backend-module-email:
+plugins/notifications-common:
+plugins/notifications-node:
 plugins/org:
-#plugins/permission-backend: ==> Added as a static plugin in RHDH
-#plugins/permission-backend-module-policy-allow-all: ==> Not meaningful for RHDH
+plugins/org-react:
+# plugins/permission-backend ==> Added as a static plugin in RHDH
+# plugins/permission-backend-module-policy-allow-all ==> Added as a static plugin in RHDH
+plugins/permission-common:
+plugins/permission-node:
+plugins/permission-react:
 plugins/proxy-backend:
-#plugins/scaffolder: ==> Added as a static plugin in RHDH
-#plugins/scaffolder-backend: ==> Added as a static plugin in RHDH
-plugins/scaffolder-backend-module-azure:
+# plugins/scaffolder  ==> Added as a static plugin in RHDH
+# plugins/scaffolder-backend  ==> Added as a static plugin in RHDH
+# plugins/scaffolder-backend-module-azure  ==> wrapper
 plugins/scaffolder-backend-module-bitbucket:
-plugins/scaffolder-backend-module-bitbucket-cloud:
-plugins/scaffolder-backend-module-bitbucket-server:
+# plugins/scaffolder-backend-module-bitbucket-cloud  ==> wrapper
+# plugins/scaffolder-backend-module-bitbucket-server  ==> wrapper
 plugins/scaffolder-backend-module-confluence-to-markdown:
 plugins/scaffolder-backend-module-cookiecutter:
-plugins/scaffolder-backend-module-gerrit:
+plugins/scaffolder-backend-module-gcp:
+# plugins/scaffolder-backend-module-gerrit  ==> wrapper
 plugins/scaffolder-backend-module-gitea:
-plugins/scaffolder-backend-module-github:
-plugins/scaffolder-backend-module-gitlab:
+# plugins/scaffolder-backend-module-github  ==> wrapper
+# plugins/scaffolder-backend-module-gitlab  ==> wrapper
+plugins/scaffolder-backend-module-notifications:
 plugins/scaffolder-backend-module-rails:
 plugins/scaffolder-backend-module-sentry:
 plugins/scaffolder-backend-module-yeoman:
-#plugins/search: ==> Added as a static plugin in RHDH
-#plugins/search-backend: ==> Added as a static plugin in RHDH
+plugins/scaffolder-common:
+plugins/scaffolder-node:
+plugins/scaffolder-node-test-utils:
+plugins/scaffolder-react:
+# plugins/search  ==> Added as a static plugin in RHDH
+# plugins/search-backend  ==> Added as a static plugin in RHDH
 plugins/search-backend-module-catalog:
 plugins/search-backend-module-elasticsearch:
 plugins/search-backend-module-explore:
 plugins/search-backend-module-pg:
 plugins/search-backend-module-stack-overflow-collator:
 plugins/search-backend-module-techdocs:
+plugins/search-backend-node:
+plugins/search-common:
+plugins/search-react:
 plugins/signals:
 plugins/signals-backend:
-plugins/techdocs:
-#plugins/techdocs-backend: ==> Error: Dynamic plugins do not support native plugins. the following native modules have been transitively detected: - cpu-features
+plugins/signals-node:
+plugins/signals-react:
+# plugins/techdocs  ==> wrapper
+plugins/techdocs-addons-test-utils:
+# plugins/techdocs-backend  ==> wrapper
 plugins/techdocs-module-addons-contrib:
```

Comment:
We should check whether it is. plugin (see the `backstage.role` value in the `package.json`.

Reply:
It is listed as `frontend-plugin-module`

Reply:
OK.
Let's keep it.